### PR TITLE
fix close release destroyObserver

### PR DESCRIPTION
**Pull request by aniude**

### CODE REVIEW

**Comment by aniude** [comment]
> Good spot, not sure what happened there. Interested to know how you spotted it. Was it causing an issue?

I found that in Chrome memory watcher, when I destroy the iframe element, this object still get reference to frame object.

File: src/iframeResizer.js [hunk 1]
```diff
@@ -68,7 +68,8 @@
       onScroll: function () {
         return true
       }
-    })
+    }),
+    destroyObserver
```
[the remark this answers is not in the document]
Comment (aniude) [reply src/iframeResizer.js]: done.

**Comment by davidjbradshaw** [comment]
Just tried to merge this but I get the following eslint errors

```bash
/Users/davidbradshaw/dev/iframe-resizer/src/iframeResizer.js
   733:5  error  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  1152:9  error  Function 'createDestroyObserver' expected no return value              consistent-return
```

**Comment by davidjbradshaw** [comment]
@aniude I think the first issue can be fixed by changing
```js
 destroyObserver && destroyObserver.disconnect()
 ```
to
```js
destroyObserver?.disconnect()
 ```

Not sure what is causing the second warning

**Comment by aniude** [comment]
> @aniude I think the first issue can be fixed by changing
> 
> ```js
>  destroyObserver && destroyObserver.disconnect()
> ```
> 
> to
> 
> ```js
> destroyObserver?.disconnect()
> ```
> 
> Not sure what is causing the second warning

I changed it. check it again.

**Comment by davidjbradshaw** [comment]
Merged into V4 branch and released as v4.3.10 Thanks for the fix.

Btw you might want to checkout the version 5 I'm currently working. Draft docs can be found at http://iframe-resizer.com. Would love any feedback on it that you might have.

**Comment by aniude** [comment]
It seems that the UI is more better. I can't wait anymore. thanks.

**Comment by aniude** [comment]
> Merged into V4 branch and released as v4.3.10 Thanks for the fix.
> 
> Btw you might want to checkout the version 5 I'm currently working. Draft docs can be found at http://iframe-resizer.com. Would love any feedback on it that you might have.

It seems that the UI is more better. I can't wait anymore. thanks.

**Pull request closed without merging**